src/settings-ui/Settings.UI/SettingsXAML/Views/AdvancedPaste.xaml

```diff
@@ -82,10 +82,10 @@
                             </tkcontrols:SettingsCard.Description>
                         </tkcontrols:SettingsCard>
                         <tkcontrols:SettingsCard
-                            x:Uid="AdvancedPaste_ShowCustomPreviewSettingsCard"
-                            HeaderIcon="{ui:FontIcon Glyph=&#xE71E;}"
+                            x:Uid="AdvancedPaste_EnableAdvancedAI"
+                            HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/Icons/SemanticKernel.png}"
                             IsEnabled="{x:Bind ViewModel.IsOpenAIEnabled, Mode=OneWay}">
-                            <ToggleSwitch IsOn="{x:Bind ViewModel.ShowCustomPreview, Mode=TwoWay}" />
+                            <ToggleSwitch IsOn="{x:Bind ViewModel.IsAdvancedAIEnabled, Mode=TwoWay}" />
                         </tkcontrols:SettingsCard>
                         <InfoBar
                             x:Uid="GPO_SettingIsManaged"
```

Comment:
Please move this between "AI enabled" setting and "Advanced AI" setting. Otherwise users can misunderstand for which setting this information is.

Reply:
yes

Reply:
@htcfreek: Done

![image](https://github.com/user-attachments/assets/007a0a7d-4688-40a6-9e6c-527fa0ba1b2f)